lib/item.rb

```diff
@@ -1,3 +1,14 @@
 class Item
+attr_reader :name, :price
+
+    def initialize (info)
+        @name = info[:name]
+        @price = info[:price]
+        #takes out all weird characters and symbols then convert to float
```

Comment:
Nice note here! You could have done the below method in the initialize function instead, but this works!